tests/models/test_hooks.py

```diff
@@ -437,10 +438,9 @@ def _predict_batch(trainer, model, batches):
     [
         {},
         # these precision plugins modify the optimization flow, so testing them explicitly
-        pytest.param(dict(gpus=1, precision=16, amp_backend="native"), marks=RunIf(min_gpus=1)),
-        pytest.param(dict(gpus=1, precision=16, amp_backend="apex"), marks=RunIf(amp_apex=True, min_gpus=1)),
+        pytest.param(dict(accelerator="gpu", devices=1, precision=16, amp_backend="native"), marks=RunIf(min_gpus=1)),
         pytest.param(
-            dict(gpus=1, precision=16, strategy="deepspeed"), marks=RunIf(deepspeed=True, min_gpus=1, standalone=True)
+            dict(accelerator="gpu", devices=1, precision=16, amp_backend="apex"), marks=RunIf(amp_apex=True, min_gpus=1)
```

Comment:
why did you get rid of strategy="deepspeed" here? this test is failing 
https://github.com/PyTorchLightning/pytorch-lightning/pull/11470/checks?check_run_id=5646992166

Reply:
ohhh. I don't think I meant to get rid of that.

Reply:
reverted the test for now